elements/geosearch/main.js

```diff
@@ -0,0 +1,375 @@
+import { LitElement, html, css } from "lit";
+import Handlebars from "handlebars";
+import proj4 from "proj4";
+import _debounce from "lodash.debounce";
+
+import { button } from "../../utils/styles/button";
+
+import styles from "./style";
+
+class EOxGeoSearch extends LitElement {
+  static get properties() {
+    return {
+      /**
+       * Internal storage of OpenCage API data after a successful API request.
+       *
+       */
+      _data: { attribute: false },
```

Comment:
Private properties should not appear in the API docs (try using `@` instead of `_` for natively private properties)

![image](https://github.com/EOX-A/EOxElements/assets/26576876/a8d5e84d-ff99-4f17-b32e-681fcafd1dc9)